README.md

```diff
@@ -24,6 +24,10 @@ The entire team should contribute with this repo, giving suggestions and making
 
 First-timer? We recommend start reading `General/Editor` and then `Process Workflow` :)
 
+## Our Coding Styleguide
+
+We don't have a specific coding styleguide. Otherwise, we use our coding styleguide from linters, that they configurations/specifications files can be found at our [linters](https://github.com/sourcelevel/linters) repository.
```

Comment:
WDYT?

```suggestion
Instead of creating a specific coding styleguide for each programming language/tool that we use, we just use linters configuration files to indicate which patterns we follow. All linters configuration files can be found in a single repository at [sourcelevel/linters](https://github.com/sourcelevel/linters).
```

Reply:
Sounds better.

Reply:
Done in 46ce7a2